### PR TITLE
chore: setup cloudbuild LINT test

### PR DESCRIPTION
**Pull request by apeabody**
* Add cloudbuild LINT test
* Addresses LINT findings (READMEs, location, etc)
* Opened issue to track INT test https://github.com/terraform-google-modules/terraform-google-module-template/issues/65

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
@apeabody
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_documentation
 The documentation needs to be regenerated. Please run `make generate_docs`. 
 ```bash 
Checking for documentation generation
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=.git' '--exclude=autogen' '--exclude=*.tfvars' /workspace/terraform-google-{{cookiecutter.module_name}}/README.md /tmp/tmp.QECVm0PWp1/generate_docs/workspace/terraform-google-{{cookiecutter.module_name}}/README.md
30,32c30,32
< |------|-------------|:----:|:-----:|:-----:|
< | bucket\_name | The name of the bucket to create | string | n/a | yes |
< | project\_id | The project ID to deploy to | string | n/a | yes |
---
> |------|-------------|------|---------|:--------:|
> | bucket\_name | The name of the bucket to create | `any` | n/a | yes |
> | project\_id | The project ID to deploy to | `any` | n/a | yes |
38c38
< | bucket\_name |  |
---
> | bucket\_name | n/a |
Error: Documentation generation has not been run, please run the
'make docker_generate_docs' command and commit the above changes.
 ``` 
- ⚠️check_terraform
 Failed Terraform check. More details below. 
 ```bash 
Running terraform fmt
╷
│
 
Error: 
Invalid attribute name
│
 
│
 
  on terraform-google-{{cookiecutter.module_name}}/examples/simple_example/outputs.tf line 19, in output "bucket_name":
│
 
  19:   value       = module.
{
{ cookiecutter.module_name|replace('-', '_') }}.bucket_name
│
 
│
 
An attribute name is required after a dot.
╵
╷
│
 
Error: 
Unsupported operator
│
 
│
 
  on terraform-google-{{cookiecutter.module_name}}/examples/simple_example/outputs.tf line 19, in output "bucket_name":
│
 
  19:   value       = module.{{ cookiecutter.module_name
|
replace('-', '_') }}.bucket_name
│
 
│
 
Bitwise operators are not supported. Did you mean boolean OR ("||")?
╵
╷
│
 
Error: 
Invalid character
│
 
│
 
  on terraform-google-{{cookiecutter.module_name}}/examples/simple_example/outputs.tf line 19, in output "bucket_name":
│
 
  19:   value       = module.{{ cookiecutter.module_name|replace(
'
-', '_') }}.bucket_name
│
 
│
 
Single quotes are not valid. Use double quotes (") to enclose strings.
╵
╷
│
 
Error: 
Invalid character
│
 
│
 
  on terraform-google-{{cookiecutter.module_name}}/examples/simple_example/outputs.tf line 19, in output "bucket_name":
│
 
  19:   value       = module.{{ cookiecutter.module_name|replace('-', 
'
_') }}.bucket_name
│
 
│
 
Single quotes are not valid. Use double quotes (") to enclose strings.
╵
Error: terraform fmt failed with exit code 2
Check the output for diffs and correct using terraform fmt <dir>
 ```

**Comment by comment-bot-dev** [comment]
@apeabody
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_documentation
 The documentation needs to be regenerated. Please run `make generate_docs`. 
 ```bash 
Checking for documentation generation
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=.git' '--exclude=autogen' '--exclude=*.tfvars' /workspace/terraform-google-{{cookiecutter.module_name}}/README.md /tmp/tmp.A2oaqZj90Z/generate_docs/workspace/terraform-google-{{cookiecutter.module_name}}/README.md
30,32c30,32
< |------|-------------|:----:|:-----:|:-----:|
< | bucket\_name | The name of the bucket to create | string | n/a | yes |
< | project\_id | The project ID to deploy to | string | n/a | yes |
---
> |------|-------------|------|---------|:--------:|
> | bucket\_name | The name of the bucket to create | `any` | n/a | yes |
> | project\_id | The project ID to deploy to | `any` | n/a | yes |
38c38
< | bucket\_name |  |
---
> | bucket\_name | n/a |
Error: Documentation generation has not been run, please run the
'make docker_generate_docs' command and commit the above changes.
 ``` 
- ⚠️check_terraform
 Failed Terraform check. More details below. 
 ```bash 
Running terraform fmt
╷
│
 
Error: 
Invalid attribute name
│
 
│
 
  on terraform-google-{{cookiecutter.module_name}}/examples/simple_example/outputs.tf line 19, in output "bucket_name":
│
 
  19:   value       = module.
{
{ cookiecutter.module_name|replace('-', '_') }}.bucket_name
│
 
│
 
An attribute name is required after a dot.
╵
╷
│
 
Error: 
Unsupported operator
│
 
│
 
  on terraform-google-{{cookiecutter.module_name}}/examples/simple_example/outputs.tf line 19, in output "bucket_name":
│
 
  19:   value       = module.{{ cookiecutter.module_name
|
replace('-', '_') }}.bucket_name
│
 
│
 
Bitwise operators are not supported. Did you mean boolean OR ("||")?
╵
╷
│
 
Error: 
Invalid character
│
 
│
 
  on terraform-google-{{cookiecutter.module_name}}/examples/simple_example/outputs.tf line 19, in output "bucket_name":
│
 
  19:   value       = module.{{ cookiecutter.module_name|replace(
'
-', '_') }}.bucket_name
│
 
│
 
Single quotes are not valid. Use double quotes (") to enclose strings.
╵
╷
│
 
Error: 
Invalid character
│
 
│
 
  on terraform-google-{{cookiecutter.module_name}}/examples/simple_example/outputs.tf line 19, in output "bucket_name":
│
 
  19:   value       = module.{{ cookiecutter.module_name|replace('-', 
'
_') }}.bucket_name
│
 
│
 
Single quotes are not valid. Use double quotes (") to enclose strings.
╵
Error: terraform fmt failed with exit code 2
Check the output for diffs and correct using terraform fmt <dir>
 ```

**Comment by comment-bot-dev** [comment]
@apeabody
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_documentation
 The documentation needs to be regenerated. Please run `make generate_docs`. 
 ```bash 
Checking for documentation generation
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=.git' '--exclude=autogen' '--exclude=*.tfvars' /workspace/terraform-google-{{cookiecutter.module_name}}/README.md /tmp/tmp.gcUiw6EWGI/generate_docs/workspace/terraform-google-{{cookiecutter.module_name}}/README.md
30,32c30,32
< |------|-------------|:----:|:-----:|:-----:|
< | bucket\_name | The name of the bucket to create | string | n/a | yes |
< | project\_id | The project ID to deploy to | string | n/a | yes |
---
> |------|-------------|------|---------|:--------:|
> | bucket\_name | The name of the bucket to create | `any` | n/a | yes |
> | project\_id | The project ID to deploy to | `any` | n/a | yes |
38c38
< | bucket\_name |  |
---
> | bucket\_name | n/a |
Error: Documentation generation has not been run, please run the
'make docker_generate_docs' command and commit the above changes.
 ``` 
- ⚠️check_terraform
 Failed Terraform check. More details below. 
 ```bash 
Running terraform fmt
╷
│
 
Error: 
Invalid attribute name
│
 
│
 
  on terraform-google-{{cookiecutter.module_name}}/examples/simple_example/outputs.tf line 19, in output "bucket_name":
│
 
  19:   value       = module.
{
{ cookiecutter.module_name|replace('-', '_') }}.bucket_name
│
 
│
 
An attribute name is required after a dot.
╵
╷
│
 
Error: 
Unsupported operator
│
 
│
 
  on terraform-google-{{cookiecutter.module_name}}/examples/simple_example/outputs.tf line 19, in output "bucket_name":
│
 
  19:   value       = module.{{ cookiecutter.module_name
|
replace('-', '_') }}.bucket_name
│
 
│
 
Bitwise operators are not supported. Did you mean boolean OR ("||")?
╵
╷
│
 
Error: 
Invalid character
│
 
│
 
  on terraform-google-{{cookiecutter.module_name}}/examples/simple_example/outputs.tf line 19, in output "bucket_name":
│
 
  19:   value       = module.{{ cookiecutter.module_name|replace(
'
-', '_') }}.bucket_name
│
 
│
 
Single quotes are not valid. Use double quotes (") to enclose strings.
╵
╷
│
 
Error: 
Invalid character
│
 
│
 
  on terraform-google-{{cookiecutter.module_name}}/examples/simple_example/outputs.tf line 19, in output "bucket_name":
│
 
  19:   value       = module.{{ cookiecutter.module_name|replace('-', 
'
_') }}.bucket_name
│
 
│
 
Single quotes are not valid. Use double quotes (") to enclose strings.
╵
Error: terraform fmt failed with exit code 2
Check the output for diffs and correct using terraform fmt <dir>
 ```

**Comment by comment-bot-dev** [comment]
@apeabody
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_documentation
 The documentation needs to be regenerated. Please run `make generate_docs`. 
 ```bash 
Checking for documentation generation
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=.git' '--exclude=autogen' '--exclude=*.tfvars' /workspace/terraform-google-{{cookiecutter.module_name}}/README.md /tmp/tmp.x6vnJoL2uB/generate_docs/workspace/terraform-google-{{cookiecutter.module_name}}/README.md
30,32c30,32
< |------|-------------|:----:|:-----:|:-----:|
< | bucket\_name | The name of the bucket to create | string | n/a | yes |
< | project\_id | The project ID to deploy to | string | n/a | yes |
---
> |------|-------------|------|---------|:--------:|
> | bucket\_name | The name of the bucket to create | `any` | n/a | yes |
> | project\_id | The project ID to deploy to | `any` | n/a | yes |
38c38
< | bucket\_name |  |
---
> | bucket\_name | n/a |
Error: Documentation generation has not been run, please run the
'make docker_generate_docs' command and commit the above changes.
 ``` 
- ⚠️check_terraform
 Failed Terraform check. More details below. 
 ```bash 
Running terraform fmt
╷
│
 
Error: 
Invalid attribute name
│
 
│
 
  on terraform-google-{{cookiecutter.module_name}}/examples/simple_example/outputs.tf line 19, in output "bucket_name":
│
 
  19:   value       = module.
{
{ cookiecutter.module_name|replace('-', '_') }}.bucket_name
│
 
│
 
An attribute name is required after a dot.
╵
╷
│
 
Error: 
Unsupported operator
│
 
│
 
  on terraform-google-{{cookiecutter.module_name}}/examples/simple_example/outputs.tf line 19, in output "bucket_name":
│
 
  19:   value       = module.{{ cookiecutter.module_name
|
replace('-', '_') }}.bucket_name
│
 
│
 
Bitwise operators are not supported. Did you mean boolean OR ("||")?
╵
╷
│
 
Error: 
Invalid character
│
 
│
 
  on terraform-google-{{cookiecutter.module_name}}/examples/simple_example/outputs.tf line 19, in output "bucket_name":
│
 
  19:   value       = module.{{ cookiecutter.module_name|replace(
'
-', '_') }}.bucket_name
│
 
│
 
Single quotes are not valid. Use double quotes (") to enclose strings.
╵
╷
│
 
Error: 
Invalid character
│
 
│
 
  on terraform-google-{{cookiecutter.module_name}}/examples/simple_example/outputs.tf line 19, in output "bucket_name":
│
 
  19:   value       = module.{{ cookiecutter.module_name|replace('-', 
'
_') }}.bucket_name
│
 
│
 
Single quotes are not valid. Use double quotes (") to enclose strings.
╵
Error: terraform fmt failed with exit code 2
Check the output for diffs and correct using terraform fmt <dir>
 ```

File: build/int.cloudbuild.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+timeout: 3600s
+steps:
+- name: 'gcr.io/cloud-foundation-cicd/cft/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
+  id: 'placeholder'
+  args: ['/bin/true']
```
Comment (bharathkkb) [inline build/int.cloudbuild.yaml:19]: IIUC this would just pass? Do we need this then?

**Review by bharathkkb — APPROVED** [review]
Thanks @apeabody

File: build/int.cloudbuild.yaml (hunk 1 above)
Comment (bharathkkb) [reply build/int.cloudbuild.yaml:19]: Just read the description 🤦

Comment (apeabody) [reply build/int.cloudbuild.yaml:19]: No worries, we can discuss later, but the no-op is (currently) required by the status checks/branch protection.

**Pull request merged**